src/service/video-manager-impl.js

```diff
@@ -1629,3 +1597,29 @@ function analyticsEvent(entry, eventType, opt_vars) {
 export function installVideoManagerForDoc(nodeOrDoc) {
   registerServiceBuilderForDoc(nodeOrDoc, 'video-manager', VideoManager);
 }
+
+/**
+ * A helper that uses feature detection to get the best InOb it can.
+ * If iframed: rootMargin is ignored unless natively supported (Chrome 81+).
+ * If not iframed: all features work properly in both polyfill and built-in.
+ *
+ * @param {function(!Array<!IntersectionObserverEntry>)} ioCallback
+ * @param {!Window} win
+ * @return {!IntersectionObserver}
+ */
+function getViewportObserver(ioCallback, win) {
+  const iframed = isIframed(win);
+  const root = /** @type {?Element} */ (iframed
+    ? /** @type {*} */ (win.document)
+    : null);
+  const rootMargin = '25%';
+
+  try {
+    return new win.IntersectionObserver(ioCallback, {
+      root,
+      rootMargin,
```

Comment:
It looks like we may also need a `treshold` set for `MIN_VISIBILITY_RATIO_FOR_AUTOPLAY`?

Reply:
I'm not sure we do. Previously, there were two paths for updating visibility:
1. Calls to `this.getIntersectionChangeEntry` which essentially manually measured / constructed the equivalent of an IntersectionObserverEntry. These had a threshold.
2. Children emitting the visibility event which was triggered by viewportCallback. These were believed outright and had no threshold. 

This PR essentially removes (1) and now only has (2) so no threshold is needed to have same functionality. I'm happy to add it in though if it is desired  behavior.

Reply:
Hmm. I read the old code a bit differently. And maybe @alanorozco could help us as well with understanding it. But here's what I'm seeing: `element.viewportCallback` eventually propagated here to the `updateVisibility`, but `updateVisibility` only considers an element visible if the threshold is satisfied. 

So, I think have an InOb with the threshold solves both branches. No?

Reply:
Notice the `/* opt_forceVisible */` param, that was always set to true via the viewportCallback path.

Reply:
Oh. I see. Missed that part. If so, ok to go w/o threshold. Or you we can confirm with @alanorozco